helm/kiam-app/templates/agent-daemonset.yaml

```diff
@@ -17,6 +17,17 @@ spec:
       hostNetwork: true
       dnsPolicy: ClusterFirstWithHostNet
       serviceAccountName: {{ .Values.agent.name }}
+      nodeSelector:
+        kubernetes.io/role: worker
```

Comment:
we need to add node selector since now the taint for noSchedule on master is ignored